File: 2942. Find Words Containing Character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> res = new ArrayList<Integer>();
        for(int i=0;i<words.length;i++){
           String word = words[i];
           if(word.indexOf(x)!=-1){
            res.add(i);
           }
        }
        return res;
    }
}

File: 2942. Find Words Containing Character.py
class Solution(object):
    def findWordsContaining(self, words, x):
        """
        :type words: List[str]
        :type x: str
        :rtype: List[int]
        """
        res=[]
        i=0
        for word in words:
            if x in word:
                res.append(i)
            i=i+1
        return res
        
